gips/0054-timeline-aggregation-protocol.md

```diff
@@ -31,11 +31,11 @@ The protocol also enables the Indexers to efficiently prove "on-chain" total sum
 
 
 # Prior Art
+The original payment system developed for the Graph protocol was called Scalar.
+TAP is a drop-in replacement for Scalar, covering the whole original Scalar functionaliy and offering much more, from aggregation of receipts to blacklisting malicious Gateways/senders.
 
-TAP is an add-on to Scalar, a high throughput and low-latency microtransaction system enabling pay-per-query service between the Indexers and Gateways.
-More info about Scalar can be found in [The Graph Foundation unveils Scalar: a microtransaction for every query](https://thegraph.com/blog/scalar/) blog post. We call the resulting combination Scalar TAP.
-
-More on the prior art can be found in `Rationale and Alternatives` section
+More info about Scalar can be found in [The Graph Foundation unveils Scalar: a microtransaction for every query](https://thegraph.com/blog/scalar/) blog post.
+More on the prior art can be found in `Rationale and Alternatives` section.
```

Comment:
It is likely there are other examples, and this was not even a proposed change of this PR, however worth noting we can nicely link such references to other sections.
```suggestion
More on the prior art can be found in [Rationale and Alternatives](#rationale-and-alternatives) section.
```